pkgs/by-name/ma/mattermost-desktop/no-git.patch

```diff
@@ -0,0 +1,25 @@
+From 2e58f7f2eec29fdeb20e82bbf24db5a5e08ba2a4 Mon Sep 17 00:00:00 2001
+From: Olli Helenius <liff@iki.fi>
+Date: Thu, 2 Jan 2025 12:42:55 +0200
+Subject: [PATCH] Use version from environment
+
+---
+ webpack.config.base.js | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/webpack.config.base.js b/webpack.config.base.js
+index 90d89c77..13d3917f 100644
+--- a/webpack.config.base.js
++++ b/webpack.config.base.js
+@@ -7,7 +7,7 @@ const path = require('path');
+ 
+ const webpack = require('webpack');
+ 
+-const VERSION = childProcess.execSync('git rev-parse --short HEAD', {cwd: __dirname}).toString();
```

Comment:
Can you do this using `substituteInPlace --replace-fail`?

Reply:
Sure, changed.